package.json

```diff
@@ -15,22 +15,24 @@
   },
   "main": "lib/kv-redis.js",
   "scripts": {
-    "test": "mocha test/integration/*.js",
+    "test": "mocha test/unit/*.js && mocha test/integration/*.js",
```

Comment:
By listing them as `mocha test/unit/*.js test/integration/*.js` won't it test in that order? You could add `--bail` to get fail-fast, as well.